pyproject.toml

```diff
@@ -7,23 +7,23 @@ authors = ["Kevin Howbrook <kevin.howbrook@torchbox.com>"]
 [tool.poetry.dependencies]
 python = "~3.8"
 django = "~4.2"
-wagtail = "~6.1"
+wagtail = "~6.2"
 psycopg2 = "~2.9"
-gunicorn = {version = "~22.0", optional = true}
+gunicorn = {version = "~23.0", optional = true}
 
 # Django packages
 django-basic-auth-ip-whitelist = "~0.5"
 django-birdbath = "^2.0.0"
-django-cors-headers = "^4.3.1"
+django-cors-headers = "^4.4.0"
 django-countries = "^7.6.1"
 django-csp = "~3.8"
-django-import-export = "^3.3.8"
```

Comment:
We don't definitely need it, I only upgraded it because it had a newer version available. Do you think it wise to revert my change and use the latest v3 (3.3.9)